packages/terra-form/src/Textarea.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+
+import './Textarea.scss';
+
+const propTypes = {
```

Comment:
The jest and nightwatch tests pass in an onChange prop for controlled and uncontrolled. Was that intended?